plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
This isn't consistent with our other naming schemes for annotations - should be lower case with dashes between significant words.